### ISSUE TITLE
Confusing colours in UI

### ISSUE THREAD

**Issue by RobJN** [mapsme/cf_audit#17]
It appears that the colours in these two pages mean different things:

[1] http://audit.osmz.ru/table/Asda-petrol-stations?all=1
[2] http://audit.osmz.ru/run/Asda-petrol-stations/594

In [1] red=tag deletion proposed by import, yellow=tag change proposed by import, green=new tag proposed by import.

In [2] green=tags proposed by import (new or changed), yellow=rejecting a proposed change and retaining an existing tag, red=rejecting a proposed change and having no tag.

Or is it that in [2] green=the current selected state based on the community result so far (with the import tag always being the lower of the two)?

**Comment by Zverik** [comment]
No, in `/run`, green means you can move the point to a better location, and blue means the location is taken from OSM and cannot be changed.

I agree that green in this case is better be changed to some non-rgb colour. Do you have any ideas?

**Comment by RobJN** [comment]
Actually I meant the colours used in the table left. But yes, the pointer green colour is again a different meaning for colour green.

**Comment by Zverik** [comment]
Right, now I get it: you're talking about tags, not objects.

Yes, there is a slight difference, which, I think, helps notice changes. In case of the `/table`, you cannot choose which value to keep, so it differentiates between new and modified values. Finding the original value is not as easy in the table view, so with yellow and red dots, you know where to look.

In the validation mode, yellow and red colours aim to attract attention to tags which values are not modified, despite being proposed by an import.
